test/integration/tags_integration_test.go

```diff
@@ -354,6 +354,59 @@ func TestIntegrationTags_Network(t *testing.T) {
 
 }
 
+func TestIntegrationTags_Subnet(t *testing.T) {
+	integrationTest(t)
+
+	createNetworkRequest := cloudscale.NetworkCreateRequest{
+		Name: networkBaseName,
```

Comment:
Make sure `AutoCreateIPV4Subnet` is False here. I'm pretty sure it's not here. If we don't do it, the subnet you're requesting might conflict with the one that will be requested a bit later (the /14).

Reply:
done